### PR TITLE
Review 'Tutum on AWS FAQ' article

### PR DESCRIPTION
**Pull request by amegianeg**
https://github.com/tutumcloud/tutum-app/issues/4941

### CODE REVIEW

**Comment by amegianeg** [comment]
Will add https://github.com/tutumcloud/tutum-app/issues/5008 in this branch too, as is related content

File: faq/tutum-on-aws-faq.md [hunk 1]
```diff
@@ -4,22 +4,40 @@ The following objects will be created automatically by Tutum when deploying a no
...
+
+* A VPC with the tag name `tutum-vpc`.
+* A subnet/set of subnets in the `tutum-vpc`. Every subnet created must be tagged with the name `tutum-subnet`
+* A VPC security group named `tutum-vpc-default`. See `Can I modify the tutum-default security group?` section below for more details. 
+
 
 ## Can I modify the tutum-default security group?
 
-You can change the ports opened in the `tutum-default` or `tutum-vpc-default` security groups as long as **ports 22/tcp (ssh), 2375/tcp (docker), 6783/tcp + 6783/udp (weave) and 48001/tcp (metrics)** are open. This is a temporary limitation that will be solved in the future.
+You can change the ports opened in the `tutum-default` or `tutum-vpc-default` security groups as long as **ports 22/tcp (ssh), 2375/tcp (docker), 6783/tcp + 6783/udp (weave)** are open. This is a temporary limitation that will be solved in the future.
```
Comment (amegianeg) [inline faq/tutum-on-aws-faq.md]: Port 22 is no longer necessary as docker upgrades are done with a container

File: faq/tutum-on-aws-faq.md [hunk 2]
```diff
@@ -4,22 +4,40 @@ The following objects will be created automatically by Tutum when deploying a no
...
+* A set of subnets, if there are no subnets already created in the VPC. Tutum will create a subnet in every AZ possible, and will leave enough CIDR space for the user to create customized subnets. Every subnet created is tagged with the name `tutum-subnet`
 * A VPC security group named `tutum-vpc-default` with all ports open by default
 
 In order for Tutum to detect these objects after they are created, it's important that **their names don't change**. Otherwise, Tutum will try to recreate them and fail due to conflicts.
 
+### Select them through UI or API
+
+This gives you full control of the resources that Tutum will use to deploy your nodes, choosing the VPC, the subnet/s and the security group. You can do it through Tutum's user interface or [API](https://docs.tutum.co/v2/api/#create-a-new-node-cluster)
+
+### Have them already created
```
Comment (fermayo) [inline faq/tutum-on-aws-faq.md]: We should discourage this as the user can select it manually now. Remove this section.

**Comment by amegianeg** [comment]
This was done weeks before the new changes: need to review it. But I wrote a full new article joining all AWS articles (it's in content_blog slack channel)

**Comment by fermayo** [comment]
OK, review it ASAP as it's going to prod soon

**Comment by amegianeg** [comment]
It still needs a review, as I got the full `How to link an AWS` article but a user told us it was not updated.

**Comment by amegianeg** [comment]
https://github.com/tutumcloud/support-docs/pull/29

**Pull request closed without merging**